### PR TITLE
CHT form properties tutorial

### PR DESCRIPTION
**Pull request by smbuthia**
Closes https://github.com/medic/cht-docs/issues/276

### CODE REVIEW

File: content/en/apps/tutorials/form-properties.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+title: "Setting Form Properties"
+linkTitle: Form Properties
+weight: 8
+description: >
+  Setting Form Properties
```
Comment (MaxDiz) [inline content/en/apps/tutorials/form-properties.md:6]:
```suggestion
 How to set form properties that contain meta information related to App forms
```

File: content/en/apps/tutorials/form-properties.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+relatedContent: >
+  apps/reference/forms/app/#formsappform_namepropertiesjson
```
Comment (MaxDiz) [inline content/en/apps/tutorials/form-properties.md:8]: delete- added to description above

File: content/en/apps/tutorials/form-properties.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+You will be adding meta-data and context to an assessment workflow that allows Community Health Workers to conduct a health assessment for children under the age of 5.
+
+## Brief Overview of Key Concepts
+
+*[Form context]({{< ref "apps/reference/forms/app#formsappform_namepropertiesjson" >}})* defines when and where the form should be available in the app.
+
+## Required Resources
+
+You should have a [functioning CHT instance with `medic-conf` installed locally]({{< ref "apps/tutorials/local-setup" >}}) and a [project folder set up]({{< ref "apps/tutorials/local-setup#3-create-and-upload-a-blank-project" >}}) already and a [built assessment form]({{< ref "apps/tutorials/app-forms" >}}).
```
Comment (MaxDiz) [inline content/en/apps/tutorials/form-properties.md:26]:
```suggestion
You should have a functioning [CHT instance with `medic-conf` installed locally]({{< ref "apps/tutorials/local-setup" >}}), completed a [project folder]({{< ref "apps/tutorials/local-setup#3-create-and-upload-a-blank-project" >}}) setup, and an [assessment form]({{< ref "apps/tutorials/app-forms" >}}).
```

File: content/en/apps/tutorials/form-properties.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+## Implementation Steps
+
+Create a new file in the same folder as your `assessment.xlsx` file and name it `assessment.properties.json`.
+
+### 1. Define the Form's Title
+
+Edit the `assessment.properties.json` file and add a `title` key with the value corresponding to what you want your title to be.
```
Comment (MaxDiz) [inline content/en/apps/tutorials/form-properties.md:34]:
```suggestion
Edit the `assessment.properties.json` file and add a `title` key with the value corresponding to the desired title.
```

File: content/en/apps/tutorials/form-properties.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+---
+
+The form properties file contains meta information related to the App form.
+
+## Purpose of the Tutorial
+
+This tutorial will take you through how to write the `<form_name>.properties.json` file.
+
+You will be adding meta-data and context to an assessment workflow that allows Community Health Workers to conduct a health assessment for children under the age of 5.
```
Comment (MaxDiz) [inline content/en/apps/tutorials/form-properties.md:18]: Can you add a sentence on the value of a user adding meta-data and context?

File: content/en/apps/tutorials/form-properties.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+
+### 4. Upload the `<form_properties>.properties.json` File
+
+Run the following command from the root folder to upload the `assessment.properties.json` file:
+
+```zsh
+medic-conf --url=https://<username>:<password>@localhost --accept-self-signed-certs upload-app-forms -- assessment
+```
+
+{{% alert title="Note" %}} Be sure to replace the values `<username>` and `<password>` with the actual username and password of your test instance. {{% /alert %}}
```
Comment (MaxDiz) [inline content/en/apps/tutorials/form-properties.md:87]: can you add a summary sentence or 2 stating the final outcome of this tutorial?

File: content/en/apps/tutorials/form-properties.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+```json
+{
+  "title": "Assessment",
+  "icon": "icon-healthcare-assessment"
+}
+```
+
+### 3. Define the Form's Context
+
+Add a `context` key in the `assessment.properties.json` file. Add an object with `person`, `place` and `expression` keys. Add the boolean value `true` for the `person` key, the boolean value `false` for the `place` key and the expression `ageInYears(contact) < 5` for the `expression` key.
```
Comment (MaxDiz) [inline content/en/apps/tutorials/form-properties.md:65]: This reads a bit like a run-on sentence so I tried to break it up a bit. Alternatively, you could use bullet points.

```suggestion
First, add a `context` key in the `assessment.properties.json` file. Next, add an object with `person`, `place` and `expression` keys. Then, add the boolean value `true` for the `person` key, the boolean value `false` for the `place` key and the expression `ageInYears(contact) < 5` for the `expression` key.
```

**Review by MaxDiz — COMMENTED** [review]
Nice job! I made a few minor changes and added a couple of comments for clarity for your review.

File: content/en/apps/tutorials/form-properties.md [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+linkTitle: Form Properties
+weight: 8
+description: >
+  Setting Form Properties
+relatedContent: >
+  apps/reference/forms/app/#formsappform_namepropertiesjson
+
+---
+
+The form properties file contains meta information related to the App form.
```
[the remark this answers is not in the document]
Comment (smbuthia) [reply content/en/apps/tutorials/form-properties.md]: Updated.

File: content/en/apps/tutorials/form-properties.md (hunk 5 above)
Comment (smbuthia) [reply content/en/apps/tutorials/form-properties.md:18]: Updated.

File: content/en/apps/tutorials/form-properties.md (hunk 6 above)
Comment (smbuthia) [reply content/en/apps/tutorials/form-properties.md:93]: Updated.

**Review by MaxDiz — APPROVED** [review]
Looks great, nice addition!

**Pull request merged**